Add link to floating UI library (#399)
ueberdosis/tiptap-docs: src/content/editor/extensions/functionality/bubble-menu.mdx

### getReferencedVirtualElement

> A callback to provide the anchor coordinates used to position the menu. Should return a [virtual element](https://floating-ui.com/docs/virtual-elements) from the [Floating UI library](https://floating-ui.com/).

Reviewer: I wouldn't call it "from the FloatingUI library" because it sounds like you need to import something.

```suggestion
A callback to provide the anchor coordinates used to position the menu. Should return a [virtual element](https://floating-ui.com/docs/virtual-elements) as expected via [Floating UI](https://floating-ui.com/).
```

Author: Thank you!!
